content/docs/2.0/concepts/scaling-jobs.md

```diff
@@ -35,8 +36,8 @@ spec:
     template:
       # describes the [job template](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/)
   pollingInterval: 30  # Optional. Default: 30 seconds
-  cooldownPeriod:  300 # Optional. Default: 300 seconds
-  minReplicaCount: 0   # Optional. Default: 0
+  SuccessfulJobsHistoryLimit: 5 # Optional. Default: 100 How many completed jobs should be kept.
+  FailedJobsHistoryLimit: 5 # Optional. Default: 100 How many failed jobs should be kept.
```

Comment:
Is this casing just in the docs or also in the codebase? Because it's not really aligned with the other properties.

```suggestion
  failedJobsHistoryLimit: 5 # Optional. Default: 100. How many failed jobs should be kept.
```

Reply:
What does the "kept" imply?

Reply:
The code has correct casing.